theme/layouts/_navigation.ejs

```diff
@@ -1,9 +1,25 @@
-<nav>
-  <ul>
-    <% for (const {title, link} of site.navigation) {%>
-      <li>
-        <a href="<%- link %>" target="_blank"><%- title %></a>
-      </li>
-    <% } %>
-  </ul>
+<select id="mobile-nav">
```

Comment:
That's another tricky and interesting part for designing for a semantic web which we will improve later on.

Mobile navigation is still a navigation item and we either need to move it into our existing `<nav>` item below, or nest it into a new `<nav>` item. Both options has it's pros and cons from semantic and designing point of view. And I'm having hard time deciding for one or another right now. So whichever option you think is best please go for it, we might revisit this when we start to enhance our menu items.